enterprise/frontend/src/metabase-enterprise/audit_app/containers/AuditApp.jsx

```diff
@@ -1,15 +1,75 @@
 /* eslint-disable react/prop-types */
-import fitViewport from "metabase/hoc/FitViewPort";
+import { useMemo } from "react";
+import { jt, t } from "ttag";
 
+import Link from "metabase/core/components/Link";
+import { useCollectionListQuery } from "metabase/common/hooks";
+import { isInstanceAnalyticsCollection } from "metabase/collections/utils";
+import fitViewport from "metabase/hoc/FitViewPort";
+import { Icon } from "metabase/core/components/Icon";
+import { Flex, Box } from "metabase/ui";
 import SidebarLayout from "../components/SidebarLayoutFixedWidth";
 import { AuditSidebar } from "../components/AuditSidebar";
+import { DeprecationNotice } from "./AuditApp.styled";
 
 const Layout = fitViewport(SidebarLayout);
 
+const DeprecationSection = () => {
+  const { data: collections } = useCollectionListQuery();
+
+  const auditCollection = useMemo(
+    () => collections?.find?.(isInstanceAnalyticsCollection),
+    [collections],
+  );
+
+  if (!auditCollection) {
+    return null;
+  }
+
+  return (
+    <DeprecationNotice>
+      <Flex align="center" px="xl" py="md">
+        <Icon name="info_filled" />
+        <Box pl="sm">
+          <Box>
+            {jt`This Audit section has been upgraded to the
+              ${(
+                <Link
+                  variant="brandBold"
+                  key="link"
+                  to={`${window.MetabaseRoot}collection/${auditCollection.id}`}
```

Comment:
I don't think I've seen us use MetabaseRoot in links before. Can this be replaced with `/` ?

Reply:
I'll double check, but I'm pretty sure we need this to make it work when metabase is hosted on subpaths